fala/apps/adviser/views.py

```diff
@@ -30,3 +35,36 @@ def get(self, request, *args, **kwargs):
 
 class AccessibilityView(TemplateView):
     template_name = "adviser/accessibility-statement.html"
+
+
+def fala_search(request):
```

Comment:
What's the rationale for mixing in a function-based view alongside our class-based views?

Reply:
Now a class based version